src/resources/postcss/tickets-admin/settings/_order_modifiers.pcss

```diff
@@ -0,0 +1,38 @@
+.tec-settings-order_modifier {
+	width: 30%;
+
+	label {
+		font-weight: bold;
+	}
+
+	input[type="date"],
+	input[type="datetime-local"],
+	input[type="datetime"],
+	input[type="email"],
+	input[type="month"],
+	input[type="number"],
+	input[type="password"],
+	input[type="search"],
+	input[type="tel"],
+	input[type="text"],
+	input[type="time"],
+	input[type="url"],
+	input[type="week"],
+	select {
+		width: 25em;
+	}
+
+	input.tec_order_modifier_amount_field {
+		width: 7em;
+	}
+
+	/* Tablet view (between 768px and 1024px) */
+	@media (max-width: 1024px) {
```

Comment:
Please build bottom-up, not top-down
100% is the base, and use min-widths. There is a reason our common styles only use min-widths. Max widths only when necessary. 

```
.tec-settings-order_modifier {
	width: 100%;

        @media (--viewport-medium) { 50% }
        @media (min-width: 1024px) { 30% } (or --viewport-full if it will work at 960px)
}
```

Reply:
@redscar can you help with that ?